apps/meteor/client/views/room/contextualBar/RoomMembers/AddUsers/ValidateMatrixInvitedUsers/ValidateMatrixIdModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { Modal, Button, Box, Icon } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import type { ComponentProps, ReactElement } from 'react';
+import React from 'react';
+
+type ValidateMatrixIdModalProps = {
+	matrixIdVerifiedStatus: Map<string, string>;
+	onClose: () => void;
+	onConfirm?: () => void;
+};
+
+const verificationStatusAsIcon = (verificationStatus: string) => {
```

Comment:
Could verification status have other value or be an empty string? That case this would return undef

Reply:
it shouldn't, since the value comes from this function

https://github.com/RocketChat/Rocket.Chat/blob/89c62679a8286ead3d433a5b16041a8f47139afb/apps/meteor/server/services/federation/infrastructure/matrix/Bridge.ts#L172-L193

which can only return those 3 values